v2/internal/util/mysql/mysql.go

```diff
@@ -111,3 +117,37 @@ func DropUser(ctx context.Context, db *sql.DB, username string) error {
 	_, err := db.ExecContext(ctx, "DROP USER IF EXISTS ?", username)
 	return err
 }
+
+func newConfig(serverAddress string, database string, port int, user string, password string) *mysql.Config {
```

Comment:
Maybe this needs a more specific name to avoid conflating this `config` with other `config` elsewhere?

```suggestion
func newMySQLConfig(
        serverAddress string, 
        database string, 
        port int, 
        user string, 
        password string) *mysql.Config {
```

Reply:
I don't think so? It's privately scoped to the `mysql` package so it won't cause any issues for other packages, and AFAIK Go's recommendation is that you not prefix the package name on the type names in the package (at least, Goland gives a warning squiggly if you do that).